Resources/config/services/product-handler.xml

```diff
@@ -15,6 +15,7 @@
         <service id="sulu_sylius_consumer.guzzle_client" class="GuzzleHttp\Client"/>
 
         <service id="Sulu\Bundle\SyliusConsumerBundle\Model\Product\Handler\Message\SynchronizeProductMessageHandler">
+            <argument type="service" id="logger"/>
```

Comment:
logger should be optional (last, nullable and on-invalid null)

if the monolog-bundle is removed the logger service is not available